vernac/declare.ml

```diff
@@ -76,14 +76,15 @@ module Info = struct
     ; udecl : UState.universe_decl
     ; scope : Locality.locality
     ; hook : Hook.t option
+    ; typing_flags : Declarations.typing_flags option
```

Comment:
We really shouldn't be adding stuff that just gets ignored.

Reply:
Oh let me fix this; should be easy.

Reply:
Moved to its own PR.